database/exip.sql

```diff
@@ -236,6 +236,8 @@ CREATE TABLE `BuyerTradingHistory` (
   `outstandingPayments` tinyint(1) DEFAULT NULL,
   `failedPayments` tinyint(1) DEFAULT NULL,
   `exporterHasTradedWithBuyer` tinyint(1) DEFAULT NULL,
+  `totalOverduePayments` int DEFAULT NULL,
+  `totalOutstandingPayments` int DEFAULT NULL,
```

Comment:
I think we should hold off on merging this PR/DB updates until we have confirmation on if one of these fields is a total or not